L1Trigger/TrackFindingTracklet/src/Tracklet.cc

```diff
@@ -644,149 +647,204 @@ void Tracklet::setFitPars(double rinvfit,
   fpgatrack_ = std::make_unique<Track>(makeTrack(l1stubs));
 }
 
-std::string Tracklet::trackfitstr() {
-  string stubid0 = "111111111";
-  string stubid1 = "111111111";
-  string stubid2 = "111111111";
-  string stubid3 = "111111111";
+const std::string Tracklet::layerstubstr(const unsigned layer) const {
+  assert(layer <= 5);
+
+  std::stringstream oss("");
+  if (!layerresid_[layer].valid())
+    oss << "0|0000000|0000000000|0000000|000000000000|000000000";
+  else {
+    if (trackIndex_ < 0 || trackIndex_ > 127) {
```

Comment:
Please avoid magic number in "trackIndex_ > 127".

Reply:
Done.